src/components/va-accordion/va-accordion.tsx

```diff
@@ -74,6 +74,21 @@ export class VaAccordion {
     }
 
     clickedItem.setAttribute('open', !prevAttr);
+
+    if (!this.isScrolledIntoView(clickedItem)) {
+      clickedItem.scrollIntoView();
+    }
+  }
+
+  isScrolledIntoView(el: Element) {
+    const elemTop = el?.getBoundingClientRect().top;
+
+    if (!elemTop && elemTop !== 0) {
+      return false;
+    }
+
+    // Only partially || completely visible elements return true
+    return elemTop >= 0 && elemTop <= window.innerHeight;
```

Comment:
I'm not sure what the a11y impact here might be, so I'm going to let @1Copenut weigh in.

Reply:
I'm happy to hear what @1Copenut has to say, and I will note that we are not changing focus so I would not envision any a11y impacts.